src/varint.ts

```diff
@@ -5,8 +5,8 @@ export default {
   deserialize
 };
 
-function serialize(int: number): Uint8Array {
-  const buff = toByteArray(int);
+function serialize(int: number | bigint): Uint8Array {
+  const buff = toByteArray(parseInt(int.toString()));
```

Comment:
> Because parseInt() returns a number, it may suffer from loss of precision if the integer represented by the string is [outside the safe range](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isSafeInteger).  


Isn't `bigIntToUint8Array` a safer approach?

Reply:
Fixed in f7957a9